### ISSUE TITLE
[GLPI 9.5.1] - Dropdwon in searchformTicket never appear [ for Group in charge of Tasks ]

### ISSUE THREAD

**Issue by zorattgi** [glpi-project/glpi#7893]
**Describe the bug**

1. When you try to select in the searchcriteriaform to add rule  with that criteria [ And, Task - Group in Charge - Is ]
the dropdown menu with group never appear on the webpage ( this page will be stuck on the search after that )

2. On the main page [ Dashboard ], you click on Group View => when you try to click on the number corresponding to the number of tasks to be performed, you suppose to be in the same situation as my first point

**Page(s) URL**

1. https://glpi/front/ticket.php

2. https://glpi/front/central.php => Tab "Group View"

If applicable, page(s) URL where the bug happens.

**To reproduce**

Go to the beginning of my post :)

**Expected behavior**

**Logs**

```
020-08-28 14:08:18] glpiphplog.CRITICAL:   *** Uncaught Exception TypeError: Argument 1 passed to Dropdown::addNewCondition() must be of the type array, string given, called in /var/www/html/glpi/inc/dropdown.class.php on line 157 in /var/www/html/glpi/inc/dropdown.class.php at line 278

```

**Screenshots**

![image](https://user-images.githubusercontent.com/40174872/91559918-346a2280-e939-11ea-949b-4fd1bcc55b68.png)

**Your GLPI setup (you can find it in Setup > General menu, System tab)**

```
 
Operating system: Linux GLPI-01 4.19.0-10-amd64 #1 SMP Debian 4.19.132-1 (2020-07-24) x86_64
PHP 7.4.8 apache2handler (Core, FFI, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apc, apcu, bz2,
	calendar, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imap, intl, json, ldap, libxml,
	mailparse, mbstring, mysqli, mysqlnd, openssl, pcre, pdo_mysql, posix, readline, session, shmop, snmp, sockets, sodium, ssh2,
	standard, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="30" memory_limit="256M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="2M" 
Software: Apache/2.4.38 (Debian) (Apache/2.4.38 (Debian) Server at glpi Port 443)
	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:68.0) Gecko/20100101 Firefox/68.0
Server Software: Debian 9.9
	Server Version: 10.1.41-MariaDB-0+deb9u1
	Server SQL Mode: 
	Parameters: glpi@localhost/glpi
	Host info: Localhost via UNIX socket

Version: GLPI 9.5.1
```

*** Workaround ***

I removed the array condition in  /var/www/html/glpi/inc/dropdown.class.php  at line 278 and it's working better but not perfectly

![image](https://user-images.githubusercontent.com/40174872/91560462-2ff23980-e93a-11ea-9c37-84e593242b6e.png)

![image](https://user-images.githubusercontent.com/40174872/91560433-27016800-e93a-11ea-8d0b-9496c4b0019a.png)

![image](https://user-images.githubusercontent.com/40174872/91560267-d853ce00-e939-11ea-9d24-3cce8576f905.png)

*** error log link to my workaround ****

[2020-08-28 14:22:43] glpiphplog.CRITICAL:   *** Uncaught Exception Error: Unsupported operand types in /var/www/html/glpi/inc/dropdown.class.php at line 2415
  Backtrace :
  ajax/getDropdownValue.php:47                       Dropdown::getDropdownValue()

**Comment by J-C-P** [comment]
I confirm the error on two different 9.5.1 instances.

**Comment by cedric-anne** [comment]
Hi,

It should be fixed by #7768 or #7878 

Regards

**Comment by zorattgi** [comment]
> 
> 
> Hi,
> 
> It should be fixed by #7768 or #7878
> 
> Regards

Thank's, 
this is the one i need to solve my issue #7768
